decryptor/postgresql/prepared_statements_sql_observer.go

```diff
@@ -71,8 +76,20 @@ func (encryptor *PreparedStatementsQuery) onPrepare(ctx context.Context, prepare
 		return nil, false, ErrStatementAlreadyInRegistry
 	}
 
-	var stmt = prepareQuery.PreparedStatementQuery.(sqlparser.Statement)
-	var preparedStatement = NewPreparedStatement(preparedStatementName, sqlparser.String(stmt), stmt)
+	var stmt = prepareQuery.GetQuery()
+	var prepareParseResult = &pg_query.ParseResult{
+		Stmts: []*pg_query.RawStmt{{
+			Stmt: stmt,
+		}},
+	}
+	stmtText, err := pg_query.Deparse(prepareParseResult)
```

Comment:
do we need statement's text or we can store statement or text as we do with query object and decide it do we need to convert to text on request?